src/Request/Criteria/LineItemFindCriteriaFactory.php

```diff
@@ -64,12 +64,12 @@ private function applyDateFilter(
         string $field,
         string $method
     ): void {
-        if ($request->get($field) !== null) {
-            $timestamp = (int) $request->get($field);
+        if ($request->query->get($field) !== null) {
+            $timestamp = (int) $request->query->get($field);
```

Comment:
if int is expected, then let's check is_int instead of == null
and I'd extract it into a variable earlier, as we use 3 times

Reply:
I think you right. But it's out of scope of this PR. This code was merged about year ago. So I have changed only necessary (for pipeline) part.

Reply:
please create a ticket then

Reply:
https://oat-sa.atlassian.net/browse/LSI-705
Done